MarijaSaysHello/Program.cs

```diff
@@ -1,8 +1,20 @@
-﻿Console.Write("Hi! What’s your name?:  ");
-var name = Console.ReadLine();
-Console.WriteLine($"Nice to meet you {name} ");
-Console.WriteLine("Marija");
+﻿//Console.Write("Hi! What’s your name?:  ");
+//var name = Console.ReadLine();
+//Console.WriteLine($"Nice to meet you {name} ");
+//Console.WriteLine("Marija");
 
-Console.WriteLine("What's the name of company where you want to work?: ");
-var companyName = Console.ReadLine();
+//Console.WriteLine("What's the name of company where you want to work?: ");
+//var companyName = Console.ReadLine();
 
+using System;
+
+namespace Marija_SaysHello 
+{
+    internal class Program 
```

Comment:
Please install in VS extension - Trailing Whitespace Visualizer - and remove the spaces in your solution
![image](https://user-images.githubusercontent.com/23722661/209964755-dcc75faa-3725-4b39-a367-2151140c1846.png)